doc: charge-rpg (threat level extra) (#36)
fariapp/fari-games: public/documents/fari-rpgs/charge-rpg.md

>
> `Supporting details` can be upgraded or downgraded based on filling the `progress` and `danger` clocks, respectively. The `danger` clock might be filled as a consequence, while the `progress` clock is filled each milestone and as the result of `long-term projects`. Ticks on the `progress` clock can be burned as momentum on a one-for-one basis.

# Threat Level Extra | Extras

**An alternate way to think adversity in Charge**

\- By othelarian

---

Sometimes, ennemies are so powerful they have a direct effect over characters, hindering their abilityto fight back.

> Threat Level is an extra about adversity and how it can affect the character's dice rolls.

Reviewer: ```suggestion
Threat Level is an extra about how adversity can affect the character's actions.
```

## What is a Threat Level?

> While adversity doesn't affect directly the character, it's ok to use position and effect. But when it has an impact, it can be less obvious on how to deal with it and make it count mechanically speaking.

Reviewer: I'm not sure I understand the messages you're trying to convey here (tu peux répondre en français qu'on se comprenne bien)

Author: I made a mistake, I was thinking "as long as" instead of "While". I will use the therabidbanana suggestion to rework this part.

> `Threat Level` is a way to represent this.

Author: I'm not sure about "powerful opposition". When I tihnk about the mist, it isn't really powerful, but it still has enough impact because it affects character's capacity.

> It takes the form of a value associated with the adversity. The higher the value, the most threatening it is.

Reviewer: ```suggestion
It takes the form of a value associated with the adversity. The higher the value, the more threatening it is.
```

## How to define a Threat Level?

A Threat Level has two main components:

* A defined source of the `Threat Level`
* A value associated with the source

> The source must be clearly identify and describe to let no room for player's interpretation. Also, the source must define how it generates the `Threat Level`.

Reviewer: ```suggestion
The source must be clearly identified so that it there's no room for the players' interpretation. Also, the source must define how it generates the `Threat Level`.
```

> #### Threat Level example: the mist
>

> > The `mist` wrap the characters in its moisty texture, overwhelming every move, hindering the perception, blunting the dexterity.

Reviewer: ```suggestion
> The `mist` wrap the characters in its damp texture, overwhelming their every move, hindering their perception, and blunting their dexterity.
```

Reviewer: yes 100%

> > A heavy `mist` is a simple threat, with a value of 1, just enough to slightly deter characters.

Reviewer: ```suggestion
> A heavy `mist` is a simple threat, with a value of 1, just enough to slightly affect the characters.
```

> > A poisonous `mist` can be lethal, and more than just hindering the senses the protection needed to survive it is a reason by itself to rise the `Threat Level` to 2.

Reviewer: ```suggestion
> A poisonous `mist` can be lethal, and more than a hindrance. The protection needed to survive is a reason in it of itself to rise the `Threat Level` to a 2.
```

Reviewer 2: As above, but "in and of" instead of "in it of", and "rise" -> "raise".

```suggestion
> A poisonous `mist` can be lethal, and more than a hindrance. The protection needed to survive is a reason in and of itself to raise the `Threat Level` to a 2.
```

Reviewer: Thanks for correction my correction xD ! Yeah this is great honestly

> #### Threat Level example: the hexer
>
> More than a simple witcher, the `hexer` is a real issue for a group of intrepid adventurer, plaguing them with hexes, slowing them.

> > The `hexer`'s spells are typically `Threat Level` applied to a single character or to the entire group, depending on the `hexer`'s power.

Reviewer: ```suggestion
> Spells made by hexers are typically a `Threat Level` applied to a single character or the entire group, depending on the `hexer`'s power.
```

Reviewer 2: Suggestion - as above, but "are typically" followed by "depending" is a bit weird phrasing, so "might be"? 

```suggestion
> The `hexer`'s spells might be `Threat Level` applied to a single character or to the entire group, depending on the `hexer`'s power.
```

### Is there a limit to the Threat Level value?

> Depends on the group, but a good range would be between 1 and 3. Keep in mind that a higher value means a greater chance to totally cancel character's roll, and so make them fail really hard.

Reviewer 2: ```suggestion
Depends on the group, but a good range would be between 1 and 3. Keep in mind that a higher value means a greater chance to totally cancel a character's roll and make them fail really hard.
```

## Rolling with the Threat Dice

> Now that the `Threat Level` is clearly defined, what to do with its value?

Reviewer: ```suggestion
Now that the `Threat Level` is clearly defined, let's go over how it has an impact on the mechanics.
```

> Well, it's easy: when a player make a roll, and if the threat is applicable to the roll (like perception with the mist), the player also roll the number of dice equal to the `Threat Level` value in the same time.

Reviewer: ```suggestion
When a player makes a roll and the threat can be tied to a reason why that roll might fail (e.g. perception with the mist example), the player also roll the number of dice equal to the `Threat Level` value at the same time.
```

Reviewer 2: As above, but "the player also roll" should probably be "the player also rolls" 

```suggestion
When a player makes a roll and the threat can be tied to a reason why that roll might fail (e.g. perception with the mist example), the player also rolls the number of dice equal to the `Threat Level` value at the same time.
```

Let's call the character's dice pool the character's dice, and the dice roll for the `Threat Level` the threat dice.

> Now, it's a bit tricky: each threat dice cancel a character's **dice of the same value**.

Reviewer: I think its pretty clear so we can rephrase to:

```suggestion
Once the two pools are rolled, each dice from the threat pool cancels a **dice of the same value** in the action pool.
```

Reviewer 2: Question, possible rephrasing - if I roll a 6 and 6 in action pool, and a 6 from threat of 1 - does it cancel one, or both?

If one:

```suggestion
Once the two pools are rolled, each dice from the threat pool cancels **a single die of the same value** in the action pool.
```

If multiple:

```suggestion
Once the two pools are rolled, each dice from the threat pool cancels **all dice of the same value** in the action pool.
```

Author: @therabidbanana only one, but it's obvious now as I read my extra again that I didn't make this clear enough.

> A little example?

Reviewer: ```suggestion
For example:
```

> > #### first roll case

Reviewer: ```suggestion
> #### Example #1
```

>

> > The character as a character's dice of 2, and the `Threat Level` has a value of 3.

Reviewer: ```suggestion
> The action pool contains 2d6, and since the `threat level` has a rating of 3, the threat pool is made of 3d6.
```

also, perhaps add an example why the treat level is of 3 ?

>
> Then, the player rolls character's dice and get 3 and 5.

> > In the same time, they rolls threat dice and get 2, 3 and 6.

Reviewer: suggestion: perhaps the GM should roll the treat pool ?

```suggestion
> At the same time, the GM rolls the treat pool and gets a 2, 3 and 6.
```

Author: Maybe in a dial, but not the GM to keep the Charge idea of only player roll.

Reviewer: Though in Charge, the GM rolls sometime (fortune rolls for example)

> The 3 on character's roll is cancel.
>
> The result is 5.

> #### second roll case
>
> Same situation, but the player rolls 2, 3 and 5 on threat dice. Then both 3 and 5 are canceled.
>

> > The result is 0, a critical fail!

Reviewer: Wont rewrite this but we could the same formatting I rewrote above

Reviewer 2: I'd further suggest that you should completely write out both the threat pool and action pool for example 2, rather than saying "same as above".

## Tune it to make it deadlier

> A first dial to turn the extra a bit harsher for the players, on a 6 on a threat dice, the 6 automatically cancel one character's die, no matter its value, and the higher.

Reviewer 2: As above but cancel => cancels
```suggestion
One way to customize this is to turn the extra so that it hits a bit harder. For example, you could make it so that if the treat pool results in a 6, the 6 automatically cancels the higest die on the action pool, no matter its value.
```

> > #### first roll case, with this dial

Reviewer 2: Again would suggest completely writing out both pools, even if the same

>
> With the threat dice result 2, 3 and 6, the 6 cancel the 5 (the highest die of the character's dice pool).
>
> The result is 2.

> And it's possible to be harsher than this by canceling every character's dice that are equal **or inferior** to the threat dice.

Reviewer: ```suggestion
Another way to chance this mechanic could be that dice that are equal or inferior to the highest dice of the treat pool are canceled.
```

Reviewer 2: As above but I think chance should be "change"

```suggestion
Another way to change this mechanic could be that dice that are equal or inferior to the highest dice of the treat pool are canceled.
```

Author: Ok, this is not what I had in mind 😅 
I need to rewrite this.
